torchio/transforms/augmentation/intensity/random_labels_to_image.py

```diff
@@ -0,0 +1,222 @@
+
+from typing import Union, Tuple, Optional, Dict, Sequence
+import torch
+from ....torchio import DATA, TypeData, TypeRangeFloat, TypeNumber, AFFINE, INTENSITY
+from ....data.subject import Subject
+from ....data.image import Image
+from .. import RandomTransform
+
+MEAN_RANGE = (0.1, 0.9)
+STD_RANGE = (0.01, 0.1)
+
+
+class RandomLabelsToImage(RandomTransform):
+    r"""Generate a random image from a binary label map or a set of
+     partial volume (PV) label maps.
+
+    Args:
+        label_key: Key designating the label map in the sample
+            that will be used to generate the new image.
+            Cannot be set at the same time as pv_label_keys.
+        pv_label_keys: Keys designating the PV label maps in the sample that
+            will be used to generate the new image.
+            Cannot be set at the same time as label_key.
+        image_key: Key to which the new volume will be saved.
+            If this key corresponds to an already existing volume, zero elements from
+            the new volume will be filled with elements of the original volume.
+        gaussian_parameters: Dictionary containing the mean and standard deviation for
+            each label. For each value :math:`v`, if a tuple
+            :math:`(a, b)` is provided then
+            :math:`v \sim \mathcal{U}(a, b)`.
+            If no value is given for a label, value from default_gaussian_parameters
+            will be used.
+        default_gaussian_parameters: Dictionary containing the default
+            mean and standard deviation.
```

Comment:
add:
"used a default value for all label that are not defined in the gaussian_parameters dictionary